packs/orion/actions/lib/actions.py

```diff
@@ -176,8 +223,10 @@ def get_ncm_transfer_results(self, transfer_id, sleep_delay=10):
         """
         ts = {}
         while True:
```

Comment:
The timeout for the action runner timeout should be enough, the process in Orion is likely to time out first.